en/peripherals/dshot.md

```diff
@@ -0,0 +1,95 @@
+# DShot ESCs
+
+DShot is an alternative ESC protocol that has several advantages over PWM or OneShot:
+- Reduced latency
```

Comment:
FYI only, I turned these around so the advantages are stated then the feature that grants it (since we're listing advantages).

Reply:
That's good.